### DISCUSSION TITLE
integration of several multiome datasets

### DISCUSSION THREAD

**Discussion by mengm5** [stuart-lab/signac#1861] [Q&A]
Hi,
I have 4 10X-multiome (scATAC+scRNA) samples.
How can I integrate them?

Workflow 1
1) merge 4 scATACs as vignette: https://stuartlab.org/signac/articles/merging
2) integrate 4 scRNAs as vignette: https://satijalab.org/seurat/articles/integration_introduction
3) joint scATACs and scRNAs as vignette: https://stuartlab.org/signac/articles/pbmc_multiomic
                                              or vignette: https://stuartlab.org/signac/articles/integrate_atac

Workflow 2
1) joint scATAC and scRNA of each sample to scMultiome as vignette: https://stuartlab.org/signac/articles/pbmc_multiomic
2) integrate 4 samples' scMultiome as vignette: https://stuartlab.org/signac/articles/integrate_atac

Which one to choose?

Best
Meng

**Comment by NadiaEdel** [comment]
Hi! I would love to know the answer to this or if you were able to figure it out.
I have 9 multiome (RNA+ATAC from same cell) datasets, and I am struggling to figure out how to process them all, ended up with the same "two workflows" as suggested by mengm5.

Any tips are much appreciated!

Thank you!

Nadia

**Comment by bapoorva** [comment]
Hi !

I would love to know this as well. In fact, my dataset a bit more complicated. I have 40 samples - 8 are multiome (scrna+scatac),22 are standalone scrna and scatac on same samples and 10 do not have atac information at all. I have finished the analysis on my scrna and i have integrated datasets that are annotated. I'm really confused on how to approach this. 

Apoorva
